Makefile

```diff
@@ -114,6 +115,14 @@ $(PROGRAM): $(GO_FILES)
 	@echo "Building: $(PROGRAM), version: $(SVERSION)..."
 	time go build -i -ldflags "-X main.program=$(PROGRAM) -X main.version=$(SVERSION)" -o $(PROGRAM) $(BUILD_FLAGS);
 
+define buildrelease
+GOOS=$(1) GOARCH=$(2) go build -ldflags "-X main.program=$(PROGRAM) -X main.version=$(SVERSION) -s -w" -o $(PROGRAM)-$(1)-$(2) $(BUILD_FLAGS);
+endef
+
+cross:
```

Comment:
does this need an entry in `.PHONY` above too?

If you can rename this to `crossbuild` or something slightly more obvious that would be awesome as well!

Reply:
Well, yes, renaming to `crossbuild` is more logical